metadeploy/labels_fr-CA.json

```diff
@@ -9,7 +9,11 @@
             "description": "tagline of product"
         },
         "description": {
-            "message": "Bienvenue dans le programme d’installation de Program Management Module (PMM)!\r\n\r\nProgram Management Module est un cadre standard pour assurer le suivi, la gestion et la livraison de programmes, de services et de livraison de services.\r\n\r\n*IMPORTANT* : Salesforce recommande de ne pas installer les mises à niveau directement dans les organisations de production. Au lieu de cela, installez d’abord Program Management Module dans l’un de vos environnements sandbox ou dans une organisation de développeurs.\r\n\r\nProgram Management Module (« PMM ») de Salesforce.org est une application extensible qui comprend des composants libres et fermés permettant de répondre aux besoins fondamentaux uniques de la gestion de programmes. Le logiciel PMM fait appel à un progiciel géré de salesforce.com, Inc. (« Salesforce ») afin d’offrir aux organisations à but non lucratif un moyen de centraliser l’information sur les programmes ce qui leur permettra de gérer et de mettre en œuvre plus efficacement leurs programmes et services.\r\n\r\nAprès avoir installé Program Management Module, consultez les ressources dans notre documentation :\r\n\r\n* [Documentation sur Program Management Module](https://powerofus.force.com/PMM_Documentation)\r\n\r\nVous avez des questions? Devenez membre du groupe de clients [Program Management Module](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHjiSAE) dans le hub Power of Us.",
+<<<<<<< HEAD
+            "message": "Bienvenue dans le programme d’installation de Program Management Module (PMM)!\r\n\r\nProgram Management Module est un cadre standard pour assurer le suivi, la gestion et la livraison de programmes, de services et de livraison de services.\r\n\r\n*IMPORTANT* : Salesforce recommande de ne pas installer les mises à niveau directement dans les organisations de production. Au lieu de cela, installez d’abord Program Management Module dans l’un de vos environnements sandbox ou dans une organisation de développeurs.\r\n\r\n*Paragraphe légal si nécessaire*\r\n\r\nAprès avoir installé Program Management Module, consultez les ressources dans notre documentation :\r\n\r\n* <Documentation sur Program Management Module>\r\n\r\nVous avez des questions? Devenez membre du groupe de clients <Program Management Module> dans le hub Power of Us.",
```

Comment:
@dmkud It looks like this merge conflict made it in here ... any idea which of these two "message" attributes are the correct one?